### DISCUSSION TITLE
Thanks for the project and multiserver mode? and frp

### DISCUSSION THREAD

**Discussion by Dunkelicht** [immense/Remotely#473] [General]
Thanks for the project, it is C# and I could learn many things inside. 

My problem is that remotely need a domain and only one domain, it takes some time to setup.

I know many one uses FRP and ultravnc, but it is not easy to set up either.

https://github.com/fatedier/frp 
https://github.com/ultravnc/UltraVNC 

Could you please check to add frp to make multiply server? Comparing teamviewer and anydesk, they could make multiply servers and connected to fastest one. So I need to make some servers, maybe one or two work as backup.

Please consider the possibility?

**Comment by stevesobol** [comment]
Interesting question. 

All connections to Remotely clients get relay through a central server you operate. I'm not sure what kind of benefit you'd get from tunneling connections to the server.
